src/fparser/two/utils.py

```diff
@@ -75,6 +75,9 @@
 from fparser.two import pattern_tools as pattern
 from fparser.common.readfortran import FortranReaderBase
 
+# A list of supported extensions to the standard(s)
```

Comment:
Please add a brief comment to describe what the "x-format" extension is.

Reply:
OK, I've added some words.